frontend/src/app/accountCreation/MfaVerify/MfaVerify.tsx

```diff
@@ -50,13 +83,14 @@ export const MfaVerify = (props: Props) => {
             className="margin-top-3 flex-align-self-end margin-left-1"
             label={"Verify"}
             type={"submit"}
-            onClick={validateCode}
+            onClick={handleSubmit}
           />
         </div>
         <Button
           className="usa-button--outline display-block margin-top-3"
           label={"Send another code"}
           type={"submit"}
+          onClick={() => {}}
```

Comment:
Is this handled in a future piece of work?

Reply:
This is related to Emma's comment above - I'm gonna go ahead and implement it as part of this PR.